newIDE/app/src/ResourcesList/ResourceUtils.js

```diff
@@ -48,12 +49,42 @@ export const getLocalResourceFullPath = (
   return resourcePath;
 };
 
-export const resourceHasValidPath = (
+export const isResourcePathinProjectFolder = (
```

Comment:
nitpicking: `isPathInProjectFolder` (actually, you check the path, you don't care about the fact it's a path used for a resource)